Saule/Http/PreprocessingDelegatingHandler.cs

```diff
@@ -62,6 +62,11 @@ public PreprocessingDelegatingHandler(JsonApiConfiguration config)
 
             var value = result.Content as ObjectContent;
 
+            if (value?.Value is Newtonsoft.Json.Linq.JObject)
```

Comment:
Shouldn't this be sufficient to just pass through json responses? I'm not sure why the changes to the formatter are needed.